README.md

```diff
@@ -65,25 +65,29 @@ sudo yum install gcc-c++ automake libtool pkgconfig boost-devel \
 To install on a FreeBSD system, use
 
 ```sh
-pkg install devel/git devel/autoconf devel/automake devel/gmake devel/libtool \
+pkg install devel/git devel/cmake
```

Comment:
I was thinking we still needed gmake, but I guess the Makefiles now work with BSD make.

I'm pretty sure the list includes everything required and nothing extra, with the possible exception of pkgconf which has been raised elsewhere